src/main/java/racingcar/controller/CarController.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.controller;
+
+import java.util.List;
+
+import racingcar.domain.Car;
+import racingcar.repository.CarRepository;
+import racingcar.service.CarService;
+import racingcar.service.RandomNumberGenerator;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class CarController {
+	private CarRepository carRepository = new CarRepository();
+	int roundCount;
+
+	public void run() {
+		while (!getCarNames()) ;
+		while (!getRoundCount()) ;
+		move(roundCount);
+	}
+
+	private boolean getCarNames() {
+		try {
+			OutputView.printCarNameRequestMsg();
+			List<String> carNames = InputView.readCarNames();
+			carNames.stream().forEach(carName -> carRepository.add(new Car(carName)));
```

Comment:
```suggestion
			carNames.stream()
			.forEach(carName -> carRepository.add(new Car(carName)));
```
stream을 사용할 때 이렇게 하면 좀 더 가독성이 좋더라고요. 
가독성에 대한 선호는 제 의견이므로, 수정하실지는 직접 결정하시면 됩니다~
저는 [공식문서](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/stream/Stream.html)를 참고해봤습니다.